sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -475,17 +473,16 @@ def test_read_from_query_unflatten_records(self):
     self.assertFalse(reader.flatten_results)
 
   def test_using_both_query_and_table_fails(self):
-    with self.assertRaises(ValueError) as exn:
+    with self.assertRaisesRegexp(
+        ValueError,
+        r'Both a BigQuery table and a query were specified. Please specify only'
+        r' one of these.'):
       beam.io.BigQuerySource(table='dataset.table', query='query')
-      self.assertEqual(exn.exception.message, 'Both a BigQuery table and a'
-                       ' query were specified. Please specify only one of '
-                       'these.')
 
   def test_using_neither_query_nor_table_fails(self):
-    with self.assertRaises(ValueError) as exn:
+    with self.assertRaisesRegexp(
+        ValueError, r'A BigQuery table or a query must be specified'):
```

Comment:
<!--new_thread; commit:4709161da347f4fa5eedbd7f3c09911b573fadac; resolved:0-->
Let's add `^` and `$`.

Reply:
done